torch/csrc/jit/tensorexpr/ir_simplifier.cpp

```diff
@@ -2950,6 +3113,40 @@ ExprPtr SimplifierUnderContext::mutate(ModPtr v) {
   return alloc<Mod>(lhs_new, rhs_new);
 }
 
+bool SimplifierUnderContext::getLoopBoundInfo(
+    const ExprPtr& expr,
+    LoopBoundInfo* loop_bound_info) {
```

Comment:
For this function, if the bound info can be saved in the form of "Bound" then the definition of "LoopBoundInfo" can be removed. 
```
return analysis::Bound(got->second.first, IRSimplifier::simplify(
      alloc<Sub>(got->second.second, immLike(got->second.second, 1))));
``` 
The definition of "LoopBoundInfo" is so similar to "Bound" in the existing code.

Reply:
I removed the `LoopBoundInfo` and use `Bound`